covalent_ui/app.py

```diff
@@ -17,38 +17,31 @@
 # FITNESS FOR A PARTICULAR PURPOSE. See the License for more details.
 #
 # Relief from the License may be granted by purchasing a commercial license.
+from __future__ import annotations
 
 import argparse
 import os
 import signal
 import sys
-from datetime import datetime
 from distutils.log import debug
 from logging import Logger
 from logging.handlers import DEFAULT_TCP_LOGGING_PORT
-from multiprocessing import Process
 from pathlib import Path
 
-import dask.config
-import networkx as nx
 import simplejson
 import tailer
-from dask.distributed import LocalCluster
 from flask import Flask, jsonify, make_response, request, send_from_directory
 from flask_cors import CORS
 from flask_socketio import SocketIO
 
-from covalent._results_manager import Result
 from covalent._results_manager import results_manager as rm
 from covalent._shared_files import logger
 from covalent._shared_files.config import get_config, set_config, update_config
 from covalent._shared_files.defaults import _DEFAULT_CONSTRAINT_VALUES
 from covalent._shared_files.util_classes import Status
 from covalent_dispatcher._db.dispatchdb import DispatchDB, encode_result
 from covalent_dispatcher._service.app import bp
-
-# Configure dask to not allow daemon workers
-dask.config.set({"distributed.worker.daemon": False})
+from covalent_dispatcher._service.app_dask import DaskCluster
```

Comment:
I'm not sure we should be importing dispatcher classes in the UI but defer to @wjcunningham7 and @mshkanth on that

Reply:
this is only because this app.py is what starts both servers; it is not technically a UI file.  In the future we might consider moving this somewhere else, but not a requirement right now.